doc/introduction/circuits.rst

```diff
@@ -25,7 +25,7 @@ Circuits
 
     circuits/glossary
 
-In Strawberry Fields, photonic quantum circuits are represented as :class:`~.Program`
+In Strawberry Fields, photonic quantum circuits are represented as :class:`~strawberryfields.Program`
```

Comment:
This is to get rid of an error message when building the docs (when module A does `from B import X`, Sphinx finds the class X in both A and B and gets confused :/ )